src/src/settings.py

```diff
@@ -45,7 +45,7 @@
     'rest_framework',
 ]
 
-MIDDLEWARE = [
+MIDDLEWARE_CLASSES = [
```

Comment:
I think for Django-1.11 it's MIDDLEWARE. Although, MIDDLEWARE_CLASSES might have backward compatibility. But why change this? I think MIDDLEWARE would also work just fine.
Also, please put a comment when you are changing something in src folder(like settings.py, urls.py, wsgi.py), since it will affect all the applications.

Reply:
I was having some problems loading the Django admin page. This error was being displayed: _'WSGIRequest' object has no attribute 'user' Django admin_; a little similar to this:  https://stackoverflow.com/questions/37949198/wsgirequest-object-has-no-attribute-user-django-admin
There I found that changing MIDDLEWARE to MIDDLEWARE_CLASSES would solve the problem without affecting in a bad way the application.

Reply:
Is it ok just commenting here the changes in src folder? @rtgdk , or do you mean somewhere else? I will have in mind next time.

Reply:
Can you please check the exact version of Django. It should be 1.11.2. Let me know.

Reply:
MIDDLEWARE was introduced in 1.10 version, so it would be better to keep MIDDLEWARE_CLASSES to your own local version. Can you please not commit this change, since it will affect all the other application?

Reply:
It is ok, I will use it just in my local version.

Reply:
I will add back agaim MIDDLEWARE.